src/nodes/agent/errors.ts

```diff
@@ -8,4 +8,22 @@ class ErrorAgentNodeIdMissing<T> extends ErrorAgent<T> {
   exitCode = sysexits.UNAVAILABLE;
 }
 
-export { ErrorAgentNodeIdMissing };
+class ErrorNodesConnectionSignalRequestVerificationFailed<
+  T,
+> extends ErrorAgent<T> {
+  static description = 'Failed to verify request message signature';
+  exitCode = sysexits.TEMPFAIL;
+}
+
+class ErrorNodesConnectionSignalRelayVerificationFailed<
+  T,
+> extends ErrorAgent<T> {
+  static description = 'Failed to verify relay message signature';
+  exitCode = sysexits.TEMPFAIL;
+}
```

Comment:
Are these really `TEMPFAIL`? I would say `UNAVAILABLE` right?

Reply:
```

 /**
   * Temporary failure, indicating something that is not really an error.
   * In sendmail, this means that a mailer (e.g.) could not create a connection,
   * and the request should be reattempted later.
   */
  TEMPFAIL: 75,

// VS
/**
   * A service is unavailable. This can occur if a support program or file
   * does not exist. This can also be used as a catchall message when
   * something you wanted to do does not work, but you do not know why.
   */
  UNAVAILABLE: 69,
```

I'm using `TEMPFAIL` because it makes more sense for a network failure. That said, it's not a network problem but a verification one. So Yeah, maybne `UNAVALIABLE` makes more sense.

Reply:
Updated.